cli/command/image/pull_test.go

```diff
@@ -73,7 +73,10 @@ func TestNewPullCommandSuccess(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		buf := new(bytes.Buffer)
-		cmd := NewPullCommand(test.NewFakeCli(&fakeClient{}, buf))
+		infoFunc := func() (types.Info, error) {
+			return types.Info{IndexServerAddress: registry.IndexServer}, nil
+		}
+		cmd := NewPullCommand(test.NewFakeCli(&fakeClient{infoFunc: infoFunc}, buf))
```

Comment:
Without this, the mock `info` endpoint returns an empty `Registry` field, which triggers the warning and causes the output not to match expectations.